lib/actionkit_connector/client.rb

```diff
@@ -41,5 +41,21 @@ def basic_auth
     def base_uri
       URI.join(host, '/rest/v1/').to_s
     end
+
+    def pager_for(uri)
+      Enumerator.new do |y|
```

Comment:
Enumerators are objects you can iterate over. So you can call `.each` on them. By returning an enumerator, the consumer can call `each` and fetch all pages of the specified resource, one at a time.